wpengine/atlas-content-modeler
# MTKA-1436 Repeating Email with Min/Max OR Exact

## Description

Add Advanced setting(s) for Email Repeatable that allows for min/max inputs of emails OR exact inputs of emails.

https://wpengine.atlassian.net/browse/MTKA-1436

## Checklist

I have:

- [x] Added Min/Max to email repeatable advanced settings
- [x] Added Exact to email repeatable advanced settings
- [x] Added ability to publish email repeatable with both Min/Max OR Exact email inputs with previously implemented validation (required, domains/subdomains, etc.)

## Testing

1. Create model with Email Repeatable that has advanced settings of Min/Max OR Exact
2. Publish using model with advanced min/max OR exact settings

**Notes**: when publishing, the end-user should only be able to add inputs that fit the criteria. Example, if the max repeatable is 5, then the end-user should not be able to add more inputs than the max. Same scenario for min and exact.

## Screenshots
<img width="516" alt="Screen Shot 2022-05-13 at 9 31 35 AM" src="https://user-images.githubusercontent.com/62450648/168306647-dc31f451-58e2-49ff-ae4d-c38dcfe90f52.png">
<img width="512" alt="Screen Shot 2022-05-13 at 9 31 17 AM" src="https://user-images.githubusercontent.com/62450648/168306651-994473df-8777-47fe-823c-890235ab7a44.png">
<img width="519" alt="Screen Shot 2022-05-13 at 9 31 46 AM" src="https://user-images.githubusercontent.com/62450648/168306655-7f31ee08-0a31-41b5-9505-895e3388c448.png">
<img width="503" alt="Screen Shot 2022-05-13 at 9 31 07 AM" src="https://user-images.githubusercontent.com/62450648/168306658-1d97aa81-2d98-4795-a6f4-eb08e822386a.png">
